torch/_utils.py

```diff
@@ -120,6 +120,36 @@ def _cuda(self, device=None, non_blocking=False, **kwargs):
             untyped_storage.copy_(self, non_blocking)
             return untyped_storage
 
+def _xpu(self, device=None, non_blocking=False, **kwargs):
```

Comment:
`_xpu`, `_hpu` and `_cuda` have the same logic.

Reply:
@guangyey, @EikanWang, @albanD : thank you for your guidance. I caught an idea and work on implementation. Hopefully will update PR tomorrow.